packages/astro/src/integration/index.ts

```diff
@@ -85,6 +85,7 @@ export const sentryAstro = (options: SentryOptions = {}): AstroIntegration => {
                       },
                     },
                     debug: options.debug ?? false,
+                    ...unstable_sentryVitePluginOptions
```

Comment:
In theory we need to spread this into all of the sub-options to be correct. Similar to how we do it here: https://github.com/getsentry/sentry-javascript/blob/defeaaf99ff6180be714ae770ce1ed0da6243676/packages/nextjs/src/config/webpackPluginOptions.ts#L63

Reply:
Done: [`15198e3` (#15638)](https://github.com/getsentry/sentry-javascript/pull/15638/commits/15198e30bd04de19245793d2aac688b5a03b044c)

Btw. I'm not sure if the nextjs package spreads into sub-options correctly. It looks to me like at the end, it overwrites them anyway: https://github.com/getsentry/sentry-javascript/blob/defeaaf99ff6180be714ae770ce1ed0da6243676/packages/nextjs/src/config/webpackPluginOptions.ts#L119

It looks like there is no test that would test a scenario with `unstable_sentryWebpackPluginOptions` set 🤔

Reply:
Oh I think you're right. Thanks for catching that. I'll fix that in a follow-up 👍